lib/index.js

```diff
@@ -3,40 +3,37 @@
 import { Range } from 'atom';
 import * as jsonlint from 'jsonlint';
 
-export default class LinterJsonLint {
+const regex = '.+?line\\s(\\d+)';
 
-  static regex = '.+?line\\s(\\d+)';
-
-  static activate() {
-    require('atom-package-deps').install();
-  }
-
-  static provideLinter() {
-    return {
-      grammarScopes: ['source.json'],
-      scope: 'file',
-      lintOnFly: true,
-      lint: editor => {
-        const path = editor.getPath();
-        const text = editor.getText();
-
-        try {
-          jsonlint.parse(text);
-        } catch (error) {
-          const message = error.message;
-          const line = Number(message.match(this.regex)[1]);
-          const column = 0;
-
-          return Promise.resolve([{
-            type: 'Error',
-            text: error.message,
-            filePath: path,
-            range: new Range([line, column], [line, column + 1])
-          }]);
-        }
+export function activate() {
+  require('atom-package-deps').install();
```

Comment:
If you specify the package name (`linter-jsonlint`) it will cut out a potential slowdown during package activation.